tests/gpu/test_gpu_torch.py

```diff
@@ -20,6 +20,7 @@
 torch = pytest.importorskip("torch")
 
 
+@pytest.mark.torch
```

Comment:
We could instead use the global variable for those whole file:

```suggestion
pytestmark = pytest.mark.torch


```

https://docs.pytest.org/en/6.2.x/reference.html#globalvar-pytestmark